packages/sx05re/emuelec/bin/emuelecRunEmu.sh

```diff
@@ -445,7 +445,7 @@ fi
 #{log_addon}#
 
 # Return to default mode
-${TBASH} setres.sh
+[[ -z "$VIDEO_EMU" ]] && ${TBASH} setres.sh
```

Comment:
Rather than re-structuring show_splash I made simple changes which should be device compatible.

Reply:
Hey, it should get called if the native_video is not set. If the native video is set it ensures it's called only once. I have not tested it properly. I can remove it if you want, it would only be a marginal difference anyway since the switch is very fast.